File: src/test/java/com/ject/studytrip/mission/fixture/UpdateMissionRequestFixture.java
package com.ject.studytrip.mission.fixture;

import com.ject.studytrip.mission.presentation.dto.request.UpdateMissionRequest;

public class UpdateMissionRequestFixture {
    private String name = null;

    public UpdateMissionRequestFixture withName(String name) {
        this.name = name;
        return this;
    }

    public UpdateMissionRequest build() {
        return new UpdateMissionRequest(name);
    }
}

File: src/test/kotlin/com/ject/studytrip/mission/fixture/UpdateMissionRequestFixture.kt
package com.ject.studytrip.mission.fixture

import com.ject.studytrip.mission.presentation.dto.request.UpdateMissionRequest

class UpdateMissionRequestFixture(
    private val name: String = "TEST 새로운 미션 이름",
) {
    fun withName(name: String): UpdateMissionRequestFixture = UpdateMissionRequestFixture(name)

    fun build(): UpdateMissionRequest = UpdateMissionRequest(name)
}
